source/includes/get-started/quickstart.c

```diff
@@ -0,0 +1,37 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int main(int argc, char *argv[])
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    char *str;
+
+    mongoc_init();
+
+    client = mongoc_client_new("<connection string>");
+    collection = mongoc_client_get_collection(client, "sample_mflix", "movies");
+
+    {
+        // Specify the query filter
+        bson_t *query = bson_new()
+            BSON_APPEND_UTF8(query, "title", "The Shawshank Redemption");
```

Comment:
I think we can condense this to:

```suggestion
        bson_t *query = BCON_NEW ("title", "The Shawshank Redemption");
```